test/adapter/json_api/parse_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'test_helper'
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        module Deserialization
+          class ParseTest < Minitest::Test
+            def setup
+              @hash = {
+                'data' => {
+                  'type' => 'photos',
+                  'id' => 'zorglub',
```

Comment:
Not sure if you replied before I wrote https://github.com/rails-api/active_model_serializers/pull/1248/files#r46101704